guides/v2.2/cloud/env/environment-vars_magento.md

```diff
@@ -14,17 +14,17 @@ Use the ADMIN variables in the following table to override credentials for the M
 
 If you want to change the values after installation, connect to your environment using ssh and use the the Magento CLI [`admin:user` command]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-admin.html) to create or edit the Magento Admin user credentials.
 
-| Variable  | Default | Description |
-| -------- | -------- | ----------- |
-| `ADMIN_USERNAME`| License Owner email address | A username for the administrative user with the ability to create other users, including administrative users.|
-|`ADMIN_EMAIL`| | Email address for the administrative user. This address is used to send password reset notifications.|
-|`ADMIN_PASSWORD`| | Password for the administrative user. When the project is created a random password is generated and an email is sent to the {{ site.data.var.ece }} License Owner. During project creation, the License Owner should have already changed the password. You might need to contact the License Owner for the updated password.|
-| `ADMIN_LOCALE`    | `en_US` | The default locale used by the Magento Admin.
+| Variable       | Default                     | Description |
+| -------------- | --------------------------- | ----------- |
+|`ADMIN_USERNAME`| License Owner email address | A username for the administrative user with the ability to create other users, including administrative users.|
+|`ADMIN_EMAIL`   |                             | Email address for the administrative user. This address is used to send password reset notifications.|
+|`ADMIN_PASSWORD`|                             | Password for the administrative user. When the project is created a random password is generated and an email is sent to the {{ site.data.var.ece }} License Owner. During project creation, the License Owner should have already changed the password. You might need to contact the License Owner for the updated password.|
+|`ADMIN_LOCALE`  | `en_US`                     | The default locale used by the Magento Admin.|
 
 ## Admin URL
 
 Use the following environment variable to secure access to your Magento Admin UI. If specified, this value overrides the default URL during installation.
 
 `ADMIN_URL`—The relative URL to access the Magento Admin UI. The default URL is `/admin`. For security reasons, Magento recommends that you change the default to a unique, custom Admin URL that is not easy to guess.
 
-If you need to change the Admin URL after installation, connect to your environment using SSH and use the Magento CLI [`magento setup:config`]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-deployment.html) command to change the URL.
+If you need to change the Admin URL after installation, use the [`magento-cloud variable:set`]({{ page.baseurl }}/cloud/env/set-variables.html) command-line tool to change the URL.
```

Comment:
To fix the broken link, do you think this would be an appropriate link to use?
https://devdocs.magento.com/guides/v2.2/cloud/before/before-setup-env-2_clone.html#change-the-magento-admin-variables

Reply:
> uld be an appropriate link to use?
> https://devdocs.magento.com/guides/v2.2/cloud/before/before-setup-env-2_clone.html#change-the-magento-admin-variables

nope

Our links are:
- https://devdocs.magento.com/guides/v2.2/cloud/env/set-variables.html
- https://devdocs.magento.com/guides/v2.3/cloud/env/set-variables.html